pontoon/machinery/templates/machinery/microsoft_terminology.jinja

```diff
@@ -1,9 +1,9 @@
 <?xml version='1.0' encoding='utf-8'?>
 <soap-env:Envelope xmlns:soap-env="http://schemas.xmlsoap.org/soap/envelope/">
     <soap-env:Header>
-        <wsa:Action xmlns:wsa="http://www.w3.org/2005/08/addressing">http://api.terminology.microsoft.com/terminology/Terminology/GetTranslations</wsa:Action>
-        <wsa:MessageID xmlns:wsa="http://www.w3.org/2005/08/addressing">urn:uuid:{{ uuid }}</wsa:MessageID>
-        <wsa:To xmlns:wsa="http://www.w3.org/2005/08/addressing">http://api.terminology.microsoft.com/Terminology.svc</wsa:To>
+        <wsa:Action xmlns:wsa="https://www.w3.org/2005/08/addressing">https://api.terminology.microsoft.com/terminology/Terminology/GetTranslations</wsa:Action>
```

Comment:
Is there documentation that says that these xml namespaces are OK to be `https`?
I only find `http` ad hoc.

Reply:
Good point! Updated.